core/iwasm/aot/aot_loader.c

```diff
@@ -487,11 +493,35 @@ load_native_symbol_section(const uint8 *buf, const uint8 *buf_end,
 
         for (i = cnt - 1; i >= 0; i--) {
             read_string(p, p_end, symbol);
-            module->native_symbol_list[i] = get_native_symbol_by_name(symbol);
-            if (module->native_symbol_list[i] == NULL) {
-                set_error_buf_v(error_buf, error_buf_size,
-                                "missing native symbol: %s", symbol);
-                goto fail;
+            if (!strncmp(symbol, "i32#", 4)) {
```

Comment:
may let *i32* be renamed *n32*. less confusion over *i* representing *integer* and *f* representing *float*.

Reply:
has renamed to "f32#"